i18n/locales/en_us.json

```
@@ -1,28 +1,63 @@
 {
-  "penTool": "Pen tool (P)",
-  "paintBucketTool": "Paint bucket tool (B)",
-  "eraserTool": "Eraser tool (E)",
-  "strokeTool": "Stroke tool (L)",
-  "rectangleTool": "Rectangle tool (R)",
-  "filledRectangleTool": "Filled Rectangle Tool (ctrl+shift+R)",
-  "circleTool": "Circle tool (C)",
-  "filledCircleTool": "Filled Circle tool (ctrl+shift+C)",
-  "rectangleSelection": "Rectangle Selection (S)",
-  "colorPicker": "Color picker (O)",
-  "primaryLeftMouseButton": "Primary - left mouse button",
-  "secondaryRightMouseButton": "Secondary - right mouse button",
-  "penSize": "Pen size at 1, 4, 8, or 16 pixels",
-  "cropTheSprite": "Crop the sprite",
-  "primaryValue": "Value",
-  "secondaryValue": "Value",
-  "swapColors": "Swap colors (X)",
+  "circleDrawingTool": "Circle tool",
+  "colorPickerDrawingTool": "Color picker",
+  "colorSwapDrawingTool": "Paint all pixels of the same color",
+  "colorSwapDrawingToolDescriptorApplyToAllLayers":  "Apply to all layers",
+  "colorSwapDrawingToolDescriptorApplyToAllFrames":"Apply to all frames",
+  "ditheringDrawingTool": "Dithering tool",
+  "eraserDrawingTool": "Eraser tool",
+  "filledCircleDrawingTool": "Filled Circle tool",
+  "filledRectangleDrawingTool": "Filled Rectangle Tool",
+  "lightenDrawingTool": "Lighten",
+  "lightenDrawingToolDescriptorDarken": "Darken",
+  "lightenDrawingToolDescriptorApplyOncePerPixel": "Apply only once per pixel",
+  "moveDrawingTool": "Move tool",
+  "moveDrawingToolDescriptorApplyToAllLayers":  "Apply to all layers",
+  "moveDrawingToolDescriptorApplyToAllFrames":"Apply to all frames",
+  "moveDrawingToolDescriptorWrapCanvas": "Wrap canvas borders",
+  "paintBucketDrawingTool": "Paint bucket tool",
+  "rectangleDrawingTool": "Rectangle tool",
+  "shapeToolDrawingDescriptorKeep1To1Ratio": "Keep 1 to 1 ratio",
+  "simplePenDrawingTool": "Pen tool",
+  "strokeDrawingTool": "Stroke tool",
+  "strokeDrawingToolDescriptorHoldShift": "Hold shift to draw straight lines",
+  "verticalMirrorPenDrawingTool": "Vertical Mirror pen",
+  "verticalMirrorPenDrawingToolDescriptorHorizAxis": "Use horizontal axis",
+  "verticalMirrorPenDrawingToolDescriptorHorizAndVert": "Use horizontal and vertical axis",
```

```suggestion
  "verticalMirrorPenDrawingToolDescriptorHorizAxis": "Use horizontal axis",
  "verticalMirrorPenDrawingToolDescriptorHorizAndVertAxis": "Use horizontal and vertical axis",
```
Adding "Axis" to the end for consistency. Please update the javascript usage as well.